Packages/com.unity.inputsystem/InputSystem/Utilities/ReadOnlyArray.cs

```diff
@@ -81,6 +81,11 @@ public int IndexOf(Predicate<TValue> predicate)
             return -1;
         }
 
+        public int IndexOf(TValue value)
+        {
+            return Array.IndexOf(m_Array, value);
```

Comment:
`ReadOnlyArray` is a slice of `m_Array`.

Reply:
Oh. You're absolutely right. I'll fix that.